fluss-client/src/test/java/com/alibaba/fluss/client/admin/FlussAdminITCase.java

```diff
@@ -563,6 +557,178 @@ void testGetServerNodes() throws Exception {
     }
 
+    @Test
+    void testCreateIllegalPartitionTable() {
+        // 1. key contains separator '$'.
+        String dbName = DEFAULT_TABLE_PATH.getDatabaseName();
+        TableDescriptor partitionedTable =
+                TableDescriptor.builder()
+                        .schema(
+                                Schema.newBuilder()
+                                        .column("id", DataTypes.STRING())
+                                        .column("name", DataTypes.STRING())
+                                        .column("age$", DataTypes.INT())
+                                        .build())
+                        .distributedBy(3, "id")
+                        .partitionedBy("age$")
+                        .build();
+        TablePath tablePath = TablePath.of(dbName, "test_create_illegal_partitioned_table_1");
+        assertThatThrownBy(() -> admin.createTable(tablePath, partitionedTable, true).get())
+                .cause()
+                .isInstanceOf(InvalidTableException.class)
+                .hasMessageContaining("Partition key should not contains separator: '$'");
+    }
+
+    @Test
+    void testAddAndDropPartitionsForStaticPartition() throws Exception {
+        String dbName = DEFAULT_TABLE_PATH.getDatabaseName();
+        TableDescriptor partitionedTable =
+                TableDescriptor.builder()
+                        .schema(
+                                Schema.newBuilder()
+                                        .column("id", DataTypes.STRING())
+                                        .column("name", DataTypes.STRING())
+                                        .column("age", DataTypes.INT())
+                                        .build())
+                        .distributedBy(3, "id")
+                        .partitionedBy("age")
+                        .build();
+        TablePath tablePath = TablePath.of(dbName, "test_add_and_drop_partitioned_table");
+        admin.createTable(tablePath, partitionedTable, true).get();
+        assertPartitionInfo(admin.listPartitionInfos(tablePath).get(), Collections.emptyList());
+
+        // add two partitions.
+        admin.addPartition(
+                        tablePath, new PartitionSpec(Collections.singletonMap("age", "10")), false)
+                .get();
+        admin.addPartition(
+                        tablePath, new PartitionSpec(Collections.singletonMap("age", "11")), false)
+                .get();
+        assertPartitionInfo(admin.listPartitionInfos(tablePath).get(), Arrays.asList("10", "11"));
+
+        // drop one partition.
+        admin.dropPartition(
+                        tablePath, new PartitionSpec(Collections.singletonMap("age", "10")), false)
+                .get();
+        assertPartitionInfo(
+                admin.listPartitionInfos(tablePath).get(), Collections.singletonList("11"));
+
+        // test add partition already exists with ignoreIfNotExists = false.
+        assertThatThrownBy(
```

Comment:
Could this also verify other exceptions like `TableNotExistException`?

Reply:
Done!